docs/examples/sequence_processing/video/video_file_list_outputs.ipynb

```diff
@@ -25,7 +25,7 @@
   },
```

Comment:
<ol><li>Can we do something about the warning? Like setting the value of the option that is changing the default explicitly?</li><li>Not sure how it looks in the notebook but it seems the figures are really big. Maybe we can limit the figure size so that they are smaller?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done